framework/src/application/schema/application_config_schema.ts

```diff
@@ -79,16 +79,13 @@ export const applicationConfigSchema = {
 				fileLogLevel: {
 					type: 'string',
 					enum: ['trace', 'debug', 'info', 'warn', 'error', 'fatal', 'none'],
-					env: 'LISK_FILE_LOG_LEVEL',
-					arg: '--log,-l',
```

Comment:
Can you refer to the issue which will replace this feature? meant passing command-line options by argument or env variables in lisk-core.

Reply:
It's in LiskHQ/lisk-core#278